main-app/app/scripts/gameboard/gameboard-controller/gameBoard.js

```diff
@@ -0,0 +1,21 @@
+angular.module('Tombola.MyModule')
+.controller('MyController', ['$scope','$q', 'Server', 'ServerTurn','FreshGame', 'CoreData','BothHumans', function ($scope,$q, server, serverTurn, freshGame, coreData, bothHumans) {
+        $scope.model=freshGame;
+        $scope.boardmodel=coreData;
+        $scope.chooseBlock = function (index){
+            if (coreData.gameBoard.charAt(index)!=="0" || coreData.currentGameState === "Win"){
```

Comment:
Better on the gameboard? current game state is a game board condition and should be on that.